app/components/public/sponsor-list.js

```diff
@@ -4,7 +4,7 @@ import { orderBy, groupBy } from 'lodash';
 const { Component, computed } = Ember;
 
 export default Component.extend({
-  sponsorsGrouped: computed('sponsors.[]', function() {
+  sponsorsGrouped: computed('sponsors', function() {
     return groupBy(orderBy(this.get('sponsors'), 'level'), 'type');
```

Comment:
Doesn't matter if the array is a zero sized array. It's probably a zero sized array because there is no data. But `toArray` is needed. Only then can you use it with `orderBy` and `groupBy`. 

And please test it with data and provide a deployment